### ISSUE TITLE
Remove casecmp vs downcase preference cop

### ISSUE THREAD

**Issue by patcon** [chef/cookstyle#27]
I'm not sure a chef run has the speed requirements to make this enforcement worth it, considering it's change in the legibility of the cookbook. I would rather not be forced :)

```
# subjective clear
if node['jira']['flavor'].downcase == 'software'
  # blah
end

# subjectively unclear
if node['jira']['flavor'].casecmp('software') == 0
  # blah
end

# subjectively unclear
if !node['jira']['flavor'].casecmp('software')
  # blah
end
```

I acknowledge that this is somewhat personal tastes, but in my mind, I'd like chef recipes to be humanly legible to folks who aren't totally in-the-know on ruby speed-based optimization idiosyncrasies that prob don't matter in the context of a chef run :)

**Comment by patcon** [comment]
https://github.com/chef/cookstyle/blob/master/config/cookstyle_base.yml#L439

**Comment by tas50** [comment]
@patcon If you want to open a PR to disable this one I'll merge that in

**Comment by tas50** [comment]
This has been fixed in 1.4.0
